### PR TITLE
Added more app window config options with docs + config parsing cleanup

### PR DESCRIPTION
**Pull request by SmilyOrg**

### CODE REVIEW

File: loom/engine/loom2d/l2dStage.cpp [hunk 1]
```diff
@@ -33,7 +33,7 @@ namespace Loom2D
 
 Stage *Stage::smMainStage = NULL;
 NativeDelegate Stage::_RenderStageDelegate;
-bool Stage::sizeDirty = true;
+bool Stage::sizeDirty = false;
```
Comment (bengarney) [inline loom/engine/loom2d/l2dStage.cpp]: Why is this change needed?

**Comment by bengarney** [comment]
One nit.

File: loom/engine/loom2d/l2dStage.cpp (hunk 1 above)
Comment (SmilyOrg) [inline loom/engine/loom2d/l2dStage.cpp]: It's because the initial sizing and stuff is now handled in `core.cpp`, so window size doesn't need to be set initially (and in fact can break options such as `maximized`).

**Comment by SmilyOrg** [comment]
Replied!

**Comment by SmilyOrg** [comment]
Rebased.
